### PR TITLE
Some tests on continuous space

### PR DESCRIPTION
**Pull request by kavir1698**
For #146

### CODE REVIEW

File: test/continuousSpace_tests.jl [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  pos = (0.5, 0.5)
+  vel = sincos(2π*rand()) .* 0.05
+  dia = 0.01
+  add_agent!(pos, model1, vel, dia)
+  @test Agents.collect_ids(DBInterface.execute(model1.space.db, "select id from tab")) == [1]
+  dbrow = DBInterface.execute(model1.space.db, "select * from tab") |> DataFrame;
+  @test dbrow[1, :a] == 0.5
+  @test dbrow[1, :b] == 0.5
+  
+  move_agent!(model1.agents[1], model1)
```
Comment (Datseris) [inline test/continuousSpace_tests.jl]:
```suggestion
  move_agent!(model1.agents[1], model1)
  @test agents[1].pos[1] != 0.5
```

**Comment by Datseris** [comment]
I've updated #146 to have a checklist. Feel free to tick of everything checked by this PR and even add more things that should be tested.

**Comment by kavir1698** [comment]
This PR should be tested and merged after #145 is merged.

**Comment by Datseris** [comment]
There is one method error: https://travis-ci.com/github/JuliaDynamics/Agents.jl/jobs/299793073#L304 
not sure if this is my fault or yours :D

**Comment by kavir1698** [comment]
Yeah, I have noticed it. I am trying to solve it.

File: src/core/continuous_space.jl [hunk 2]
```diff
@@ -132,27 +132,27 @@ function biggest_id(model::ABM{A, <: ContinuousSpace}) where {A}
...
-  !isnothing(model.space.extend) && (pos .*= model.space.extend)
+  pos = Tuple(rand(space.D))
+  !isnothing(space.extend) && (pos = pos .* space.extend)
   return pos
 end
 
-function add_agent!(agent::A, model::ABM{A, <: ContinuousSpace}) where {A}
-  agent.pos = randompos(model.space)
+function add_agent!(agent::A, model::ABM{A, <: ContinuousSpace}) where {A<:AbstractAgent}
+  # agent.pos = randompos(model.space) # Why not use the agent's position?
```
Comment (Datseris) [inline src/core/continuous_space.jl]: Please read the public API of `agent_pos!`. The function that uses the agent's position is `add_agent_pos!`

Comment (Datseris) [reply src/core/continuous_space.jl]: (this change has to be reverted for consistent API)

**Comment by Datseris** [comment]
https://travis-ci.com/github/JuliaDynamics/Agents.jl/jobs/299817501#L304

**Pull request merged**